lib/vmdb/appliance.rb

```diff
@@ -90,7 +90,7 @@ def self.log_server_identity
       startup_fname = File.join(Rails.root, "log/last_startup.txt")
       FileUtils.rm_f(startup_fname) if File.exist?(startup_fname)
       begin
-        startup = VMDBLogger.new(startup_fname)
+        startup = ManageIQ::Loggers::Base.new(startup_fname)
```

Comment:
This feels wrong.  I would expect an instance of a concrete subclass, not the base.  cc @bdunne

Reply:
I opened https://github.com/ManageIQ/manageiq-loggers/issues/20 on this.  For now, this is fine, though we may want to change manageiq-loggers first and release a new version.